processes/fulfillURLManifest.py

```diff
@@ -62,31 +62,37 @@ def update_manifest(self, metadataObject, bucketName, currKey):
 
         metadataJSON = json.loads(metadataObject['Body'].read().decode("utf-8"))
         metadataJSONCopy = copy.deepcopy(metadataJSON)
+
+        #Checking if record asscociated with manifest is public domain or in copyright
```

Comment:
I think if we make this method clearer we can remove this comment. Your comment is explaining what the code is doing but your code is not explaining what it itself is doing. Lean into the latter.

Reply:
I mentioned changing the name of the method to be more clear in a earlier comment so I'll delete this comment.